great_expectations/data_context/types/base.py

```diff
@@ -1625,13 +1641,9 @@ def run_name_template(self, value: str):
         self._run_name_template = value
 
     @property
-    def batch_request(self):
+    def batch_request(self) -> Union[dict, BatchRequest]:
```

Comment:
Since type of `batch_request` is defined as Optional[dict] in __init__, shouldn't this be -> dict?

Reply:
Fixed in a better way than I had before.